ConcernsCaseWork/Service.TRAMS.Tests/Teams/TeamsServiceTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿using AutoFixture;
+using AutoFixture.AutoMoq;
+using AutoFixture.Idioms;
+using ConcernsCaseWork.Shared.Tests.Factory;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Moq.Protected;
+using NUnit.Framework;
+using Service.TRAMS.Base;
+using Service.TRAMS.Teams;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Service.TRAMS.Tests.Teams
+{
+	public class TeamsServiceTests
+	{
+		[Test]
+		public void Methods_GuardAgainstNullArgs()
+		{
+			var fixture = new AutoFixture.Fixture();
+			fixture.Customize(new AutoMoqCustomization());
+			var assertion = fixture.Create<GuardClauseAssertion>();
+			assertion.Verify(typeof(TeamsService).GetMethods());
+		}
+
+		[Test]
+		public void Constructors_GuardAgainstNullArgs()
+		{
+			var fixture = new AutoFixture.Fixture();
+			fixture.Customize(new AutoMoqCustomization());
+			var assertion = fixture.Create<GuardClauseAssertion>();
+			assertion.Verify(typeof(TeamsService).GetConstructors());
+		}
+
+		[Test]
+		public async Task GetTeam_Returns_Dto()
+		{
+			var expectedDto = new ConcernsCaseworkTeamDto("user.one", new[] { "user.two" });
```

Comment:
Again, a super small issue. Should we be using // arrange // act // assert? I don't mind either way, but nice to be consistent.